src/components/postgresql/Privileges.ts

```diff
@@ -125,7 +125,10 @@ export class Privileges extends pulumi.ComponentResource {
     } = args
 
     this.defaultPrivileges = (Object.keys(privileges) as Array<ObjectType>)
-      .filter(objectType => objectType !== 'database')
+      .filter(
+        objectType => objectType !== 'database'
+        && objectType !== 'schema',
+      )
```

Comment:
weird indentation & readability, I didn't see at first it was a function

```suggestion
      .filter(objectType => (
        objectType !== 'database'
        && objectType !== 'schema'
      ))
```